site/labs/comparisons/comparisons.rst

```diff
@@ -0,0 +1,90 @@
+****************
+Comparing Values
+****************
+
+* Feel free to use your laptop
```

Comment:
Did you ever look into it including text dynamically was possible? I feel like we chatted about it years back. This preamble is in every lab, even across courses. Would be nice to have it in one spot, or at least one per course.